app/views/profiles/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="row">
+  <div class="medium-4 columns">
+    <div>
```

Comment:
I'm confused about this comment. It seems like you are suggesting that I should remove lines 1 and 2 here, but then two comments below you suggest that I keep line 2 and change the class name. Am I misunderstanding? :confused: